playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -97,6 +104,7 @@
   register: shipyard_create_config
   failed_when:
     - shipyard_create_config.stdout.find('Error') == 0
+    - shipyard_create_config.stdout.find('Traceback') == 0
```

Comment:
good catch here... in what cases will shipyard use traceback?

Reply:
In this particular case, the Traceback was a PermissionError trying to read treasuremap.yml